kodi_addon_checker/check_artwork.py

```diff
@@ -51,7 +51,8 @@ def _check_image_type(report: Report, image_type: str, parsed_xml, addon_path: s
                     elif image_type == "fanart":
                         _check_fanart(report, width, height)
                     else:
-                        # screenshots have no size definitions
+                        if has_transparency(im):
+                            report.add(Record(PROBLEM, "screenshot.png should be solid. It has transparency."))
```

Comment:
An addon can define multiple screenshots and only one have a transparency problem. Please report the correct file name instead of a generic `screenshot.png`

Reply:
Icon and fanart is shown while browsing through addons in kodi. Screenshots are shown in the info screen of each addon.

Reply:
![image](https://user-images.githubusercontent.com/5943908/44364210-ab58bf00-a4c6-11e8-8ba4-38afcddc4dde.png)